GOVERNANCE.md

```diff
@@ -1,133 +1,315 @@
-  your goal is to become a senior maintainer that will represent your organization.
-* You should likely have already been commenting on pull requests and issues with the intent of solving
-  user problems and improving the overall quality of the project.
-* We will expect you to start contributing increasingly complicated PRs, under the guidance
-  of the existing senior maintainers.
+This document defines governance policies for the entire Crossplane project hosted and managed by
+the [Crossplane organization](https://github.com/crossplane/).
+
+## Steering Committee
+
+Steering committee members demonstrate a strong commitment to the project with views in the interest
+of the broader Crossplane community. They are the stewards of the entire Crossplane organization and
+are expected to dedicate thoughtful and serious effort towards the goal of general success in the
+ecosystem.
+
+Responsibilities include:
+
+* Own the overall charter, vision, and direction of the Crossplane project
+* Promote broad adoption of the overall project, stewarding the growth and health of the entire
+  community
+* Provide guidance for the project maintainers
+* Review and approve central/core architecture and design changes that have broad impact across
+  multiple repositories
+* Participate in the conflict resolution and voting process at the organization scope when necessary
+* Approving new repositories, or archiving repositories in the Crossplane organization
+* Add and remove members in the Crossplane organization
+* Actively participate in the regularly scheduled steering committee meetings
+* Regularly attend the recurring community meetings
+
+The current list of steering committee members is published and updated in [OWNERS.md](OWNERS.md).
+All steering committee members will be made owners of the Crossplane organization in GitHub.
+
+### Bootstrapping the steering committee
+
+The membership of the steering committee will go through an initial bootstrapping process to help
+transition the stewardship and leadership of the community over a clearly defined time period and
+process while ensuring that the project remains true to its initial charter.  It is essential that
+the existing vision and strategy continues to guide the project while it continues to grow.
+Therefore, transition in stewardship should not happen abruptly and the founding leadership should
+not be diluted too quickly while the project is still being bootstrapped and transitioned.
+
+Therefore, the membership will be bootstrapped over a **2 year period**, from the date of acceptance
+of this governance, as follows:
+
+* Membership will be **limited to 5** total seats
+* **3 seats** are granted to the senior maintainers from the existing Crossplane governance: Bassam
+    Tabbara, Jared Watts, and Nic Cope
+  * **2 year** term
+* **2 seats** to be filled by members of the community selected by the above committee members
```

Comment:
we are in the process of confirming members for these two seats and will update this PR once they have been confirmed